recipe/meta.yaml

```diff
@@ -31,13 +31,13 @@ requirements:
   build:
     - python    # [win]
     - cmake >=3.1
-    - zlib
+    - zlib 1.2*
   run:
-    - zlib
+    - zlib 1.2*
 
 test:
   requires:
-    - python    # [win]
+    - python {{ environ['PY_VER'] + '*' }}  # [win]
```

Comment:
Though, I should add, we haven't needed to pin Python previously here as it has pulled in the right one because of the `zlib` dependendency. That being said, I'm ok with this change. 👍